htdocs/modulebuilder/template/admin/myobject_extrafields.php

```diff
@@ -28,27 +29,25 @@
 
 // Load Dolibarr environment
 $res = 0;
+// Try main.inc.php into web root defined in DOCUMENT_ROOT, very frequently defined
```

Comment:
DOCUMENT_ROOT is not defined with the realpath of files and can differs from the real path that we need to have into the conf.php file for different purposes.
This is why we prefer CONTEX_DOCUMENT_ROOT.
Not as often defined but when defined, value is always good. For DOCUMENT_ROOT, value can be good or wrong and we have no way to make the difference.